woocommerce-services.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Plugin Name: WooCommerce Services
+ * Plugin Name: WooCommerce Services (deprecated)
```

Comment:
Maybe keep here `Connect for WooCommerce (deprecated)` to make this transition less confusing?